sdks/java/io/hadoop/jdk1.8-tests/src/test/resources/kubernetes/cassandra/LargeITCluster/data-load.sh

```diff
@@ -20,6 +20,27 @@
 
 set -e
 recordcount=1000
+
+# Delete cassandra single node set up if already exists
+services_exists="$(kubectl get svc -o=name)"
+#convert string of space(' ') separated words to array of words
+service_list=( $services_exists )
+
+function delete_service {
+cd ../LargeITCluster
```

Comment:
what I mean by this is: please indent a little if that's the norm for bash scripts